File: represent/bench/c_flat/FormalBench_0235__FormalBenchHF__diverse__rule_1__IsOctagonal_llm_2.c
#include <stdlib.h>

int isOctagonal(int num);

int isOctagonal(int num) {

		int *arr = (int *)malloc(sizeof(int) * (num + 1));
int arr_len = num + 1;
		arr[0] = 0;
		for (int c = 0; c <= num - 1; c++) {
			arr[c + 1] = arr[c] + 6 * c + 1;
		}
		return arr[num];
}

File: represent/bench/java_flat/FormalBench_0235__FormalBenchHF__diverse__rule_1__IsOctagonal_llm_2.java
import java.io.*;
import java.lang.*;
import java.util.*;
import java.math.*;

class IsOctagonal {

	public static int isOctagonal(int num) {
		int[] arr = new int[num + 1];
		arr[0] = 0;
		for (int c = 0; c <= num - 1; c++) {
			arr[c + 1] = arr[c] + 6 * c + 1;
		}
		return arr[num];
	}
}
